utils/generate_snowball.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+"""Refresh and update language stemming data from the Snowball project."""
+
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#     "requests>=2.30",
+# ]
+# ///
+
+from __future__ import annotations
+
+import shutil
+import subprocess
+import sys
+import tarfile
+import tempfile
+from io import BytesIO
+from pathlib import Path
+
+import requests
+
+SNOWBALL_VERSION = '3.0.1'
+SNOWBALL_URL = f'https://github.com/snowballstem/snowball/archive/refs/tags/v{SNOWBALL_VERSION}.tar.gz'
+
+ROOT = Path(__file__).resolve().parent.parent
+SEARCH_DIR = ROOT / 'sphinx' / 'search'
+STOPWORDS_DIR = SEARCH_DIR / '_stopwords'
+NON_MINIFIED_JS_DIR = SEARCH_DIR / 'non-minified-js'
+
+STOPWORD_URLS = (
```

Comment:
@ojwb are stopwords for multiple languages available as a combined download, or do we need to collect each file individually (as here)?

Reply:
(apologies; I've only just noticed that this somewhat-duplicates previous discussion.  even so, it would be convenient..)

Reply:
That seems a good idea - I'd just thought about the comment stripping aspect in the other thread, but being able to grab all the lists in one download would be handy.

Reply:
Thank you - and FWIW: the specific use-case I have in mind for this is to allow adding checksums for the stopwords files (similar to [another comment](https://github.com/sphinx-doc/sphinx/pull/13561#discussion_r2093514285) I left about the `snowball` tarball download).

Assuming that updates to those files would occur similarly to localization/translation files, e.g. they may occur piecemeal and in somewhat unpredictable order, but can generally be bundled and approved for a release version, then inluding a snapshot of those in a versioned file could be convenient.